IntegerManipulationPractise/Shuffle.cs

```diff
@@ -10,13 +10,35 @@ namespace IntegerManipulationPractise
     class Shuffle
     {
         //Take list of numbers, shuffle it, return it...
-        public static List<int> ShuffleList(List<int> list)
+        public static List<int> ShuffleListRandomInsert(List<int> list)
         {
+            var recivedList = list;
             var retVal = new List<int>();
+            var random = new Random();
+            foreach (var element in recivedList)
+            {
+                retVal.Insert(random.Next(0, retVal.Count+1), element);//mora +1 zato sto nikada nece prebaciti prvi broj...
```

Comment:
to sam sebi konju napisao komentar, posto sam radi testove i testove i prvi broj mi je uvek bio na kraju, i onda sam shvatio da sam imbecil koji nije dobro izracunao:)